oracle/metrics/server.go

```diff
@@ -29,7 +30,7 @@ type PrometheusServer struct {
 	logger *zap.Logger
 }
 
-// StartPrometheusServer creates a prometheus server if the metrics are enabled and
+// NewPrometheusServer creates a prometheus server if the metrics are enabled and
```

Comment:
i forgot to delete this server, this is duplicate and should be removed since the new prom server is now in service/servers/prometheus

Reply:
Do we need anything in this file?

Reply:
na

Reply:
its all copy paste